config/config.go

```diff
@@ -79,43 +80,66 @@ func DefaultConfig() Config {
 	return Config{Hazelcast: hz}
 }
 
-const defaultUserConfig = `
-hazelcast:
-  cluster:
-    name: dev
-    unisocket: true
-    network:
-      # 0s is no timeout
-      connectiontimeout: 0s
-      addresses:
-      - "localhost:5701"
-    cloud:
-      token: ""
-      enabled: false
-    security:
-      credentials:
-        username: ""
-        password: ""
-    discovery:
-      usepublicip: false
-  logger:
-    level: error
-ssl:
-  enabled: false
-  servername: ""
-  capath: ""
-  certpath: ""
-  keypath: ""
-  keypassword: ""
-# disables auto completion on interactive mode if true
-noautocompletion: false
-styling:
-  # builtin themes: default, no-color, solarized
-  theme: "default"
-`
+type UserFriendlyConfig struct {
+	Hazelcast struct {
+		Cluster struct {
+			Name      string `yaml:"name"`
+			UniSocket bool   `yaml:"unisocket"`
+			Network   struct {
+				ConnectionTimeout string   `yaml:"connectiontimeout"`
```

Comment:
Is there a reason to explicitly provide YAML field name as in yaml:"connectiontimeout"?

Reply:
This struct is removed. See the new version https://github.com/hazelcast/hazelcast-commandline-client/pull/139/commits/b0d972e6837094463a0188aad274170a47d0af6a.